### DISCUSSION TITLE
Activate Other Data Source on Self-Hosted Docker

### DISCUSSION THREAD

**Discussion by BreoganGal** [ghostfolio/ghostfolio#2915] [Ideas]
Good evening, first of all, thank you very much to the dev for all the work he does in this magnificent job.

In this case I come to propose an idea that someone has probably already suggested before and that is **that there is an option in the configuration panel to change the data source**, since yahoo does not work very well in certain altcoins, for example IOTA (which It hasn't been updated for a long time).

In case you see this suggestion as something absurd, could you provide some guide for "mortals" so that anyone can edit this field without extensive knowledge of code.

Regards, and thank you very much.

Various threads where they talk about it:
https://github.com/ghostfolio/ghostfolio/discussions/2606
https://github.com/ghostfolio/ghostfolio/discussions/2090

**Comment by dtslvr** [comment]
Hi @BreoganGal

Thank you for your feedback.

> since yahoo does not work very well in certain altcoins

For this, Ghostfolio uses _CoinGecko_ which is an officially supported data source and activated by default. Just pick the search result with the symbol `iota`. You can get an overview of your assets with their corresponding data sources in _Admin Control_ panel > _Market Data_.
